diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -92,6 +92,17 @@ open class DiktatJavaExecTaskBase @Inject constructor(
             project.logger.info("Setting system property for diktat config to $it")
         })
         args = additionalFlags.toMutableList().apply {
+            /*
+             * Disable the standard rules via the command line.
+             */
+            run {
+                val ktlintRuleSetIds = sequenceOf("standard", "experimental", "test", "custom")
```

Comment:
Is this still required if those artifacts are excluded from dependencies?

Reply:
No it isn't (basically, dependency exclusion is sufficient), but I'd rather leave this **on** as a safety net.

Reply:
To avoid confusion, I've expanded the comment with 44570bba3fd825688e83c9dd9c582eaa19c2552e.